src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{

```

Comment:
There should be a space between the class name and the opening brace to conform with Java coding standards.